FileSpecs/CharData.md

```diff
@@ -62,13 +64,94 @@ Offset | Type | Name | Description
 0092 | Word | Current food |
 0094 | Word | Defense | Seems to be used by players only? Maybe each byte has a meaning like variable and base defense?
 0096 | Word | Damage | Seems to be used by players only? Maybe each byte has a meaning like variable and base damage?
+00CE | Word | Experience Points
```

Comment:
As mentioned below, I would think this is a 4 byte value starting at 00CC. Otherwise it would be limited to 65535 exp. At least in Ambermoon you can have more than 100000 exp easily.